### PR TITLE
Exceptions to Sentry basics

### PR DESCRIPTION
**Pull request by steveburkett**
Connect Event Capture API to Sentry 

https://app.zenhub.com/workspaces/quasar-5ef270981e560b0019803111/issues/openstax/quasar/23

image

### CODE REVIEW

File: app/controllers/api/v0/base_controller.rb [hunk 1]
```diff
@@ -2,6 +2,10 @@ class Api::V0::BaseController < ApplicationController
   include Swagger::Blocks
   include OpenStax::Swagger::Bind
 
+  rescue_from_unless_local StandardError, send_to_sentry: true do |ex|
+    render json: binding_error(status_code: 500, messages: [ex.message]), status: 500
+  end
```
Comment (reedstrm) [inline app/controllers/api/v0/base_controller.rb:7]: So this turns any exception into a 500 when sending to sentry. Are there other useful statuses we might want to use?

**Review by reedstrm — COMMENTED** [review]
I think this is deep enough in Rails magic that I want another set of eyes to say "yes". It seems to make sense to me, though.

File: app/controllers/api/v0/base_controller.rb (hunk 1 above)
Comment (steveburkett) [reply app/controllers/api/v0/base_controller.rb:7]: @reedstrm there will be.  this is just a start.

**Review by reedstrm — APPROVED** [review]

**Comment by reedstrm** [comment]
I went and studied what Tutor did for this, and compared, and decided I do know enough now to say yes. :)

**Pull request merged**